datasets/tne/README.md

```diff
@@ -0,0 +1,186 @@
+---
+annotations_creators:
+- crowdsourced
+language_creators:
+- found
+languages:
+- en
+licenses:
+- mit
+multilinguality:
+- monolingual
+pretty_name: Text-based NP Enrichment
+size_categories:
+- unknown
```

Comment:
```suggestion
- 1K<n<10K
```

Reply:
Can't believe I missed that.
Btw, it's a bit more nuanced than that. We have in the dataset about 5K documents, but there are many more annotations. What is the standard here? To report the number of documents or annotations?

Reply:
We report the number of samples (i.e. the elements that we `yield`), which here correspond to the documents